Clinical-Genomics/scout
# Populate variants filter chromosomes choices according to genome build of the case

fix #2481

Default chromosomes list contains chromosome "MT" and whenever this is selected in the variants filters page it doesn't return any results for cases in genome build 38 (because its variants have chromosome = "M")

This PR moves the code that populate the choices of the chromosomes outside the Form and into the controllers, so that the chromosome choices reflect the case genome build.

**How to test**:
1. It would be ideal to have one case in genome build 38 and one in build 37.
2. Check that the form chromosomes select contains values and the filtering by chromosome work 

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
